spec.html

```diff
@@ -11760,6 +11789,10 @@ <h1>Static Semantics: PropName</h1>
         <emu-alg>
           1. Return StringValue of |IdentifierReference|.
         </emu-alg>
+        <emu-grammar>PropertyDefinition : `...` AssignmentExpression</emu-grammar>
+        <emu-alg>
+          1. Return ~empty~.
+        </emu-alg>
```

Comment:
Hopefully this satisfies https://github.com/tc39/proposal-object-rest-spread/issues/42 @anba